cloud/aws/templates/aws_oidc/bin/aws_template.py

```diff
@@ -22,3 +23,11 @@ def _tf_run_for_aws(self, is_destroy=False):
     def _setup_shared_state_file(self):
         if self.config.use_backend_config():
             backend_setup.setup_backend_config(self.config)
+
+    def setup_log_file(self):
+        # TODO(#2606): If remote file exist fetch it here.
+        _, self.log_file_path = tempfile.mkstemp()
```

Comment:
Pass a prefix to the `mkstemp` so that this file can be identified? Also printing its name to console would be useful for later debugging.

Reply:
done.